R/palettes.R

```diff
@@ -327,7 +327,7 @@ palettes <- lapply(
       gradient = c(low='#f0f0f0', high='#092639')
     ),
     
-    camoflauge = list(
+    camouflage = list(
```

Comment:
I think we should create a changelog / `NEWS` file and note this breaking change.